docs/source/levels/intermediate_level_12.rst

```diff
@@ -0,0 +1,44 @@
+############################
+Level 12: Deploy your models
+############################
+
+In this level you'll learn a few options for deploying models into production.
+
+----
+
+.. raw:: html
+
+    <div class="display-card-container">
+        <div class="row">
+
+.. Add callout items below this line
+
+.. displayitem::
+   :header: Deploy with ONNX
+   :description: Optimize models for enterprise-scale production environments with ONNX.
+   :col_css: col-md-4
+   :button_link: ../deploy/production_advanced.html
+   :height: 180
+   :tag: advanced
+
+.. displayitem::
+   :header: Deploy with torchscript
+   :description: Optimize models for enterprise-scale production environments with torchscript.
+   :col_css: col-md-4
+   :button_link: ../deploy/production_advanced_2.html
+   :height: 180
+   :tag: advanced
+
+.. displayitem::
+   :header: Compress models for fast inference
+   :description: Compress model size for deployment with Quantization and Pruning.
```

Comment:
Should quantization and pruning be capitalized? Maybe we should use italics for emphasis?